### ISSUE TITLE
[COGNITION] Facility detail second-level lists should be nested properly for assistive devices

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#484]
## Issue
The facility detail views sometimes have second-level lists that use CSS to indent them instead of nesting an `<ul>` inside the `<li>`. These lists were flagged as SC 1.3.1 issues. Screenshots attached below.

## Audit Finding
* Note 1, Defect 1

## Related Issue
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/485

## Acceptance Criteria
* As an assistive device user, I want to hear the lists announced as proper nested lists, instead of one large list.
* As a user with a level of visual acuity, I do not want to see any visual changes in the application.

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/facility/vha_674GD

## Possible Fixes (optional)
This will need to include some HTML restructuring. Basic pseudo-code with no CSS consideration included here:

```html
<ul>
  <li>Primary Care: <strong>19 days</strong></li>
  <li>Specialty care:
    <ul>
      <li class="sublist">Cardiology: <strong>7 days</strong></li>
      <li class="sublist">Dermatology: <strong>5 days</strong></li>
      <li class="sublist">Mental Health: <strong>23 days</strong></li>
    </ul>
  </li>
</ul>
```

## WCAG or Vendor Guidance (optional)
* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs
![Screen Shot 2019-07-03 at 5.27.35 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/be558ce1-754b-40b2-b97c-bbf280eae2fa)

**Comment by aeltarhoni** [comment]
Related ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/360

**Comment by Karak888** [comment]
@1Copenut @alexpappasoddball would you kindly validate / review these changes that @kelsonic made? Thanks!

**Comment by 1Copenut** [comment]
@Karak888 && @kelsonic I reviewed this in Staging using the URL https://staging.va.gov/find-locations/facility/vha_674GD from the original description. The axe error is still being reported because the UL is not nested inside the LI. I've attached a screenshot below. 

Let me know if you want to discuss over video chat and I'll follow up late today or Monday morning.

---
![Screen Shot 2019-09-20 at 12 01 37 PM](https://user-images.githubusercontent.com/934879/65345078-031c9a80-db9f-11e9-95a2-1bca4a95b09a.png)

**Comment by kelsonic** [comment]
@1Copenut I haven't used the Axe extension before, but it looks very handy! I see the issue as well and will send up a follow-up PR where the below issue(s) are solved :) Thanks!

![image](https://user-images.githubusercontent.com/12773166/65345444-47f8ff00-dba8-11e9-9665-69c9e7b3450f.png)

**Comment by 1Copenut** [comment]
@kelsonic Glad to hear axe is helpful. I encourage all front-end teammates to download it for their preferred Chrome/Firefox browser if they're allowed to do so. It's helpful in finding and verifying issues.

**Comment by kelsonic** [comment]
@1Copenut @alexpappasoddball Ready for another round of validation! Thank you! :)

**Comment by 1Copenut** [comment]
@kelsonic This looks good for facilities. I apologize for not raising a Community Care Provider in the original ticket too. If you check out https://staging.va.gov/find-locations/provider/ccp_1851813299 and run axe, you'll see this error appear again.

If you make the same fix you've made for this last go-round, we'll be 💯.

**Comment by kelsonic** [comment]
@1Copenut No worries! Do you mind if I create a separate ticket/branch for that?

**Comment by Karak888** [comment]
@kelsonic I will close this one and if you haven't already please create a new one for community care.  Thanks!

**Comment by kelsonic** [comment]
New issue created for Community Care: https://github.com/department-of-veterans-affairs/va.gov-team/issues/2015 👍
